server/controllers/imageUpload.js

```diff
@@ -0,0 +1,35 @@
+
+const imageUrl = require("../models/imageUpload");
+const asyncHandler = require('express-async-handler');
+const util = require('util');
+const { cloudinary } = require('../utils/cloudinary');
+
+// @route POST /upload
+// @desc insert user profile
+// @access Private
+exports.uploadPicture = asyncHandler(async (req, res, next) => {
+ 
+    const userProfile = await Profile.findOne({ userId: req.user.id });
+
+    if(!userProfile){
+      res.status(404).send({
+        message: "user profile doesn't exist "});
+    }
+
+    const fileString = req.body.files;
+
+    try{
+        const uploadedReponse = await cloudinary.uploader.upload(fileString, {
+            upload_preset: 'ml_default'
+        })
+        imageUrl.create({uploadedReponse});
```

Comment:
There is something inconsistent here. Your model has only a single property, named `imageUrlPath`. You're passing create an object with a single property `uploadedResponse`. 

Upload response contains a lot of properties ([here](https://cloudinary.com/documentation/image_upload_api_reference#upload_response) is an example). I suggest you identify the props you need from the example (e.g. `url/secure_url`) and only pass those to your model.